ann_benchmarks/plotting/metrics.py

```diff
@@ -73,6 +73,31 @@ def rel(dataset_distances, run_distances, metrics):
         print("Found cached result")
     return metrics.attrs['rel']
 
+
+def rel(dataset_distances, run_distances, metrics):
+    if 'rel' not in metrics:
+        print('Computing rel metrics')
+        rel_group = metrics.create_group('rel')
+
+        k = run_distances.shape[1]
+        # Take the first k distances
+        dataset_distances = dataset_distances[:,:k]
+        dataset_sum       = np.sum(dataset_distances, axis=1)
+        run_sum           = np.sum(run_distances, axis=1)
+
+        relative_errors = run_sum / dataset_sum
+
+        rel_group.attrs['mean'] = np.mean(relative_errors)
+        rel_group.attrs['std'] = np.std(relative_errors)
+        percentiles = [5,25,50,75,95]
+        percentile_values = np.percentile(relative_errors, percentiles)
+        for p, v in zip(percentiles, percentile_values):
+            rel_group.attrs['perc-' + str(p)] = v
+        rel_group['relative_errors'] = relative_errors
+    else:
+        print("Found cached result")
+    return metrics['rel']
+
```

Comment:
It seems to me that we duplicate a lot of code with these percentile calculations, at some point we should refactor.